docker/perf_layer_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Enable the performance layers by setting the evironment variables.
+# Run vkcube and make sure it runs without fault when performance layers are 
+# enabled.
+# Test if each layer write to its corresponding output file. 
+
+set -e
+
+# Create a temp output directory for log files. 
+readonly OUTPUT_DIR="$(mktemp -d)"
+readonly INSTALL_DIR="$1"
+readonly LAYER_DIR="$2"
+
+declare -a output_files
+output_files=("compile_time.csv" "run_time.csv" "memory_usage.csv" "frame_time.csv")
+
+#######################################
+# Check if the layers write data in their specified log files.
+# Globals:
+#   OUTPUT_DIR
+# Arguments:
+#   layer's log file name
+# Outputs:
+#   If data does not exist in the file, write an error message to stdout and
+#   exit. 
+#######################################
+check_layer_log() {
+  local file_name="$1"
+
+  if [[ $(wc -l <"${OUTPUT_DIR}"/"${file_name}") -lt 2 ]]; then
+    echo "Error: ${file_name} has not been populated properly." 
+    exit 1
+  fi
+}
+
+export LD_LIBRARY_PATH="${INSTALL_DIR}":$LD_LIBRARY_PATH
+export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time
+export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime
+export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_frame_time
+export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage
+export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload
+export VK_LAYER_PATH="${LAYER_DIR}"
+export VK_COMPILE_TIME_LOG="${OUTPUT_DIR}"/compile_time.csv
+export VK_RUNTIME_LOG="${OUTPUT_DIR}"/run_time.csv
+export VK_FRAME_TIME_LOG="${OUTPUT_DIR}"/frame_time.csv
+export VK_MEMORY_USAGE_LOG="${OUTPUT_DIR}"/memory_usage.csv
+export VK_PIPELINE_CACHE_SIDELOAD_FILE="${OUTPUT_DIR}"/cache_sideload_file.csv
+export VK_PERFORMANCE_LAYERS_EVENT_LOG_FILE="${OUTPUT_DIR}"/events.log
+
+# Create the input file for cache sideload pipeline.
+touch "${OUTPUT_DIR}"/cache_sideload_file.csv
```

Comment:
This is won't be a a valid pipeline cache file. According to the Vulkan specs, pipeline cache blob has to contain a header. So if we really wanted to pass some input, we could create something like a 256-byte binary file filled with all zeros, e.g., using `dd`. But I think it would also be fine not to give it any input, because the driver will reject it anyway.